modules/ROOT/pages/api-extensions.adoc

```diff
@@ -54,26 +54,21 @@ In order to do so, edit `custom-permissions-mapping.properties` to give the perm
 [NOTE]
 ====
 
-REST API permissions are stored in the user's session and new permissions will only be effective for a user the next time they log into Bonita Portal.
+REST API permissions are stored in the user's session and new permissions will only be effective for a user the next time they log into the Bonita Applications.
```

Comment:
We don't use `the` in other places
```suggestion
REST API permissions are stored in the user's session and new permissions will only be effective for a user the next time they log into Bonita Applications.
```

Reply:
I think we should have "log into the Bonita Runtime" instead of "log into the Bonita Applications"  since we log into the runtime and not into the apps.

Reply:
Either way, I would keep the "the", since we talk about a specific thing (the Bonita Applications).